src/functions-and-arrays.js

```diff
@@ -1,41 +1,112 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
 
+function maxOfTwoNumbers(num1, num2) {
+  let bigNumber;
+
+  if (num1 > num2) {
+      bigNumber = num1;
+  } else {
+      bigNumber = num2;
+  }
+
+  return bigNumber;
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+    if (words.length === 0) {
+      return null;
+    }
+
+    if (words.length === 1) {
+      return words[0]
+    }
+
+    let longestWord = '';
+
+        for (let i = 0; i < words.length; i++) {
+            const findWord = words[i];
+
+            if (findWord.length > longestWord.length || longestWord === '') {
+                longestWord = findWord;
+            }
+
+        }
+
+    return longestWord;
+}
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(numbers) {
```

Comment:
```suggestion
function sum(numbers) {
```
Para que no te falle, el nombre de las funciones tiene que ser el que te venia por defecto si no falla los test de jasmine jejeje

Reply:
ahora mismo el codigo que tienes implementado solo suma numeros, es decir si te llega un string, o un array, o un objeto, o un booleano los intentas sumar todos, y eso no deberia ser asi, tienes que especificarlos un poco, te pongo como lo haria yo vale? y me dices cualquier cosa y lo vemos vale??